mods/action.js

```diff
@@ -148,6 +148,17 @@ function buildEditResponse(res) {
     return res;
 }
 
+function buildUserInfoResponse(res) {
```

Comment:
User info should be handled differently. Firstly, it's part of the `query` action, so having an endpoint in the same level as `query` is not good conceptually. So, eventually we could change `/query` endpoint to become `/query{/meta}` and then check to see if that param is specified, otherwise use the one provided by the user in the body.

Secondly, we should probably handle all of the query returns in `buildQueryResponse`. For now, a simple check to see if it's about user info and then check the object would do, but ideally, the same error checks apply to all query metas - response code check, basic body check, etc, so we would have a general check block and then, depending on the meta, use different routines to do further checks and/or response rewrites.